packages/slice-machine/src/legacy/lib/models/common/widgets/Link/components.tsx

```diff
@@ -95,3 +89,41 @@ export function RepeatableCheckbox(props: CommonCheckboxProps) {
     </Box>
   );
 }
+
+export function Variant() {
+  return (
+    <Box overflow="hidden" flexDirection="column" border borderRadius={6}>
+      <Box
+        border={{ bottom: true }}
+        padding={12}
+        flexDirection="column"
+        gap={8}
+      >
+        <Text variant="h4" color="grey12">
+          Variants
+        </Text>
+        <Text color="grey11">
+          Add variants like "Primary" and "Secondary" to allow editors to choose
```

Comment:
💡 I also prefer your new copy, I let you confirm it with Côme for both sentences and all good for me.